src/Data/ServiceObject/LocalStorage/localStorageServiceObjectDataClass.ts

```diff
@@ -720,11 +720,28 @@ async function attributes(): Promise<DataClassSchema> {
       { title: lang === 'de' ? 'Modellnummer' : 'Model number' },
     ],
     number: ['string', { title: lang === 'de' ? 'Nummer' : 'Number' }],
+    parentId: [
+      'reference',
+      {
+        title:
+          lang === 'de'
+            ? 'Übergeordnetes Serviceobjekt'
+            : 'Parent service object',
+        to: 'ServiceObject',
+      },
+    ],
     product: [
       'string',
       { title: lang === 'de' ? 'Produktname' : 'Product name' },
     ],
     productType,
+    responsibleAgent: [
+      'reference',
+      {
+        title: lang === 'de' ? 'Responsible agent' : 'Verantwortlicher',
```

Comment:
* [ ] `de`/`en` mixed up
* [ ] FYI Inconsistent wording: in other places we use `responsibleAgent: "Zuständiger Bearbeiter"`.